<launch>


    <!-- CubicPolynomialSampingPlanner -->
    <node pkg="cubic_polynomial_planner" type="local_planner" name="localPlanner" output="screen" required = "true">
    <!-- <rosparam file="$(find CubicPolynomialSampingPlanner)/launch/lattice_planner_para.yaml" 
    # command="load" ns="lattice_planner" />
    -->

	    <param name="r_circle" value="1.0" type="double"/>
	    <param name="d_circle" value="2.0" type="double" />
	    <param name="obstacle_inflation" value="1.5" type="double" />
	    <param name="alpha1" value="100.0" type="double"/>
	    <param name="alpha2" value="1.0" type="double" />
	    <param name="alpha3" value="10.0" type="double" />
	    <param name="alpha4" value="0.0" type="double" />
	    <param name="lane_width" value="3.75" type="double"/>
	    <param name="longitudinal_num" value="5" type="int"/>
	    <param name="lateral_num" value="15" type="int"/>
	    <param name="longitudinal_step" value="20.0" type="double" />
	    <param name="lateral_step" value="0.5" type="double"/>
	    
	    <param name="s0" value="1.0" type="double"/>
	    <param name="startRho" value="-1.0" type="double"/>
	    <param name="startHeading" value="0.0" type="double"/>
	    <param name="showLatticeFlag" value="true" type="bool"/>


    </node>
    <!-- 运行rviz -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find cubic_polynomial_planner)/launch/config.rviz"/>

</launch>
